src/it/java/org/owasp/webgoat/CSRFIntegrationTest.java

```diff
@@ -23,7 +23,7 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.DynamicTest.dynamicTest;
 
-public class CSRFTest extends IntegrationTest {
+public class CSRFIntegrationTest extends IntegrationTest {
```

Comment:
Like the same that has been done in the previous one class (AccessControlIntegrationTest), and we're using JUnit5, 
I would like to modify the visibility to _default_ of this one and also their methods. 
It applies to the rest of the classes that have been modified on this PR.